app/controllers/api/v1/user_blocks_controller.rb

```diff
@@ -0,0 +1,51 @@
+class Api::V1::UserBlocksController < ApplicationController
+  skip_before_action :verify_authenticity_token
+  
+  def get_blocked_users
```

Comment:
Can you move these two methods to a helper file or the User model and then use them here, instead of defining them in the controller?